grid-client/deployer/tf_plugin_client.go

```diff
@@ -47,11 +47,11 @@ var (
 		"main": "https://graphql.grid.tf/graphql",
 	}
 	// RelayURLS relay urls
-	RelayURLS = map[string]string{
-		"dev":  "wss://relay.dev.grid.tf",
-		"test": "wss://relay.test.grid.tf",
-		"qa":   "wss://relay.qa.grid.tf",
-		"main": "wss://relay.grid.tf",
+	RelayURLS = map[string][]string{
```

Comment:
Use the same ones defined here 
https://github.com/threefoldtech/zos/blob/main/pkg/environment/environment.go#L103